docs/language/functions.rst

```diff
@@ -384,6 +384,11 @@ is executed. This made clear in the declarations; ``receive()`` must be declared
 with value and no ``receive()`` function is defined, then the call reverts, likewise if
 call is made without value and no ``fallback()`` is defined, then the call also reverts.
 
+The fallback function can defined in two ways. First, it can have no parameters or return
+values. Alternatively, it must have a ``bytes`` parameter and ``bytes`` return value. In this
+case, the parameter contains the undecoded input (also known as calldata or instruction data),
```

Comment:
undecoded = encoded?

Reply:
The wording is not great. What I mean is the raw bytes data before going through borsh/scale/eth decoding.